proposals/cordova8.md

```diff
@@ -0,0 +1,53 @@
+We have come with a few ideas for the next major version of Cordova. 
+
+### Goals
+
+* Reduce maintenance burden of cordova tooling
+* Refactor cordova-lib 
+* Improve unit test coverage
+    
+### Proposed Changes
+
+* Remove non cordova-fetch fetching code
+    * remove no-fetch flag + any associated files with non cordova-fetch fetching
+    * drop npm dependency
+    * issue: https://issues.apache.org/jira/browse/CB-13055
+
+* Create new `cordova install` command
+    * remove restore functionality from `cordova prepare` command
+    * move restore functionality into new `cordova install` command. (installing plugins/platforms listed in `package.json` and `config.xml`)
+    * update docs to include `cordova install` 
+    * issue: https://issues.apache.org/jira/browse/CB-11981
+    * proposal: https://github.com/cordova/cordova-discuss/pull/54
+
+* Remove plugins dir from cordova projects
+This directory has traditionally been a staging directory. We copy plugins from the `plugins` directory to `platforms/platform/...` when a plugin needs to get installed. Now, these plugin directories also live in `node_modules` directory. I propose we delete the `plugins` directory and instead just copy the plugins straight from `node_modules`.
+    * TODO: investigate reason why `fetch.json`, + `<platform>.json` exist and if we need them. They live in plugins directory right now.
+    * https://issues.apache.org/jira/browse/CB-13059
+
+* Remove `cordova platform save` command
+This command isn't necessary anymore since we save by default since `cordova@7`. It was previously used to save added plugins and platforms to `config.xml`. 
+    * **Do Now** add deprecation notice for next `cordova@7` release.
```

Comment:
deprecation notice for platform save
https://github.com/apache/cordova-lib/pull/585/files